### PR TITLE
update build-in-alert doc

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/monitor-built-in-alerting.md [hunk 1]
```diff
@@ -52,15 +52,19 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| Data migration job met error during data export | Please check the error and refer to [Troubleshoot DM](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshootin) for help.  |
+| Data migration job met error during data import | Please check the error and refer to [Troubleshoot DM](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshootin) for help. |
+| Data migration job met error during incremental data migration | Please check the error and refer to [Troubleshoot DM](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshootin) for help. |
+| Data migration job has been paused for more than 6 hours during incremental migration | Data migration job has been paused for more than 6 hours during incremental migration, binlog in upstream database might be purged(depends on your database binlog purge strategy) and may cause incremental migration fail. Please refer to [Troubleshoot DM](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshootin) for help. |
```
Comment (Frank945946) [inline tidb-cloud/monitor-built-in-alerting.md:65]: may cause incremental migration to fail

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/monitor-built-in-alerting.md [hunk 2]
```diff
@@ -52,15 +52,19 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| TiKV storage utilization exceeds 80% | Total TiKV storage utilization of cluster ABC in project XYZ exceeds 80%. Consider increasing the node number or node storage size for TiKV to increase your storage capacity. To monitor storage utilization, see [Monitoring metrics](/tidb-cloud/monitor-tidb-cluster.md#monitoring-metrics). |
+| TiFlash storage utilization exceeds 80% | Total TiFlash storage utilization of cluster ABC in project XYZ exceeds 80%. Consider increasing the node number or node storage size for TiFlash to increase your storage capacity. To monitor storage utilization, see [Monitoring metrics](/tidb-cloud/monitor-tidb-cluster.md#monitoring-metrics). |
+| One or more cluster nodes are offline | Some or all nodes in cluster ABC in project XYZ are offline. The TiDB Cloud Operations team is aware and working to resolve the issue. Refer to [TiDB Cloud Status](https://status.tidbcloud.com/) for the latest information. To monitor node status, see [Cluster status and node status](/tidb-cloud/monitor-tidb-cluster.md#cluster-status-and-node-status). `This alert will be removed from 2023/3/31.`  |
```
Comment (TomShawn) [inline tidb-cloud/monitor-built-in-alerting.md:61]: We don't make predictions in technical documentation. See [Document future features  |  Google developer documentation style guide  |  Google Developers](https://developers.google.com/style/future).
```suggestion
| One or more cluster nodes are offline | Some or all nodes in cluster ABC in project XYZ are offline. The TiDB Cloud Operations team is aware and working to resolve the issue. Refer to [TiDB Cloud Status](https://status.tidbcloud.com/) for the latest information. To monitor node status, see [Cluster status and node status](/tidb-cloud/monitor-tidb-cluster.md#cluster-status-and-node-status). This alert will be removed in a future release.  |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 3]
```diff
@@ -52,15 +52,19 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| Data migration job met error during data export | Please check the error and refer to [Troubleshoot DM](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshootin) for help.  |
+| Data migration job met error during data import | Please check the error and refer to [Troubleshoot DM](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshootin) for help. |
+| Data migration job met error during incremental data migration | Please check the error and refer to [Troubleshoot DM](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshootin) for help. |
```
Comment (TomShawn) [inline tidb-cloud/monitor-built-in-alerting.md:64]:
```suggestion
| Data migration job meets error during data export | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md) for help.  |
| Data migration job meets error during data import | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md) for help. |
| Data migration job meets error during incremental data migration | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md) for help. |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 4]
```diff
@@ -52,15 +52,19 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| Data migration job has been paused for more than 6 hours during incremental migration | Data migration job has been paused for more than 6 hours during incremental migration, binlog in upstream database might be purged(depends on your database binlog purge strategy) and may cause incremental migration to fail. Please refer to [Troubleshoot DM](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshootin) for help. |
```
Comment (TomShawn) [inline tidb-cloud/monitor-built-in-alerting.md:65]:
```suggestion
| Data migration job has been paused for more than 6 hours during incremental migration | Data migration job has been paused for more than 6 hours during data incremental migration. The binlog in the upstream database might be purged (depending on your database binlog purge strategy) and might cause incremental migration to fail. See [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md) for help. |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 5]
```diff
@@ -52,15 +52,19 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| Replication lag is larger than 10 minutes and still increasing for more than 20 minutes | Please refer to [Troubleshoot DM](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshootin) for help. |
```
Comment (TomShawn) [inline tidb-cloud/monitor-built-in-alerting.md:66]:
```suggestion
| Replication lag is larger than 10 minutes and still increasing for more than 20 minutes | See [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md) for help. |
```

File: tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md [hunk 6]
```diff
@@ -94,10 +94,15 @@ You can subscribe to TiDB Cloud alert emails to be informed in time when an aler
 The following are alerts about Data Migration: 
 
 - "Data migration job met error during data export"
+    - Recommended Action : Check the error message from data migration page, and refer to [Migration errors and solutions](#migration-errors-and-solutions) for help.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md:97]:
```suggestion

    Recommended action: check the error message on the data migration page, and see [Migration errors and solutions](#migration-errors-and-solutions) for help.

```

File: tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md [hunk 7]
```diff
@@ -94,10 +94,15 @@ You can subscribe to TiDB Cloud alert emails to be informed in time when an aler
...
 - "Data migration job met error during data import"
+    - Recommended Action : Check the error message from data migration page, and refer to [Migration errors and solutions](#migration-errors-and-solutions) for help.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md:99]:
```suggestion

    Recommended action: check the error message on the data migration page, and see [Migration errors and solutions](#migration-errors-and-solutions) for help.

```

File: tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md [hunk 8]
```diff
@@ -94,10 +94,15 @@ You can subscribe to TiDB Cloud alert emails to be informed in time when an aler
...
 - "Data migration job met error during incremental data migration"
-- "Data migration job has been paused for more than 6 hours during incremental migration" (If this alert occurs, you can resume the data migration job or ignore this alert directly.)      
+    - Recommended Action : Check the error message from data migration page, and refer to [Migration errors and solutions](#migration-errors-and-solutions) for help.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md:101]:
```suggestion

    Recommended action: check the error message on the data migration page, and see [Migration errors and solutions](#migration-errors-and-solutions) for help.

```

File: tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md [hunk 9]
```diff
@@ -94,10 +94,15 @@ You can subscribe to TiDB Cloud alert emails to be informed in time when an aler
...
+- "Data migration job has been paused for more than 6 hours during incremental migration" 
+    - Recommended Action : If this alert occurs, you can resume the data migration job or ignore this alert directly.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md:103]:
```suggestion

    Recommended action: resume the data migration job or ignore this alert.     

```

File: tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md [hunk 10]
```diff
@@ -94,10 +94,15 @@ You can subscribe to TiDB Cloud alert emails to be informed in time when an aler
...
 - "Data migration job met error during data import"
+    - Recommended Action : Check the error message from data migration page, and refer to [Migration errors and solutions](#migration-errors-and-solutions) for help.
 - "Data migration job met error during incremental data migration"
-- "Data migration job has been paused for more than 6 hours during incremental migration" (If this alert occurs, you can resume the data migration job or ignore this alert directly.)      
+    - Recommended Action : Check the error message from data migration page, and refer to [Migration errors and solutions](#migration-errors-and-solutions) for help.
+- "Data migration job has been paused for more than 6 hours during incremental migration" 
+    - Recommended Action : If this alert occurs, you can resume the data migration job or ignore this alert directly.     
 - "Replication lag is larger than 10 minutes and stilling increasing for more than 20 minutes"
+    - Recommended Action : Contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for consultation.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md:105]:
```suggestion

    - Recommended action: contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for help.

```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 11]
```diff
@@ -52,15 +52,19 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| Data migration job met error during data export | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md) for help.  |
+| Data migration job met error during data import | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md) for help. |
+| Data migration job met error during incremental migration | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md) for help. |
+| Data migration job has been paused for more than 6 hours during incremental migration | Data migration job has been paused for more than 6 hours during data incremental migration. The binlog in the upstream database might be purged (depending on your database binlog purge strategy) and might cause incremental migration to fail. See [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md) for help. |
+| Replication lag is larger than 10 minutes and still increasing for more than 20 minutes | See [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md) for help. |
```
Comment (TomShawn) [inline tidb-cloud/monitor-built-in-alerting.md:66]:
```suggestion
| Data migration job met error during data export | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help.  |
| Data migration job met error during data import | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
| Data migration job met error during incremental migration | Check the error and see [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
| Data migration job has been paused for more than 6 hours during incremental migration | Data migration job has been paused for more than 6 hours during data incremental migration. The binlog in the upstream database might be purged (depending on your database binlog purge strategy) and might cause incremental migration to fail. See [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
| Replication lag is larger than 10 minutes and still increasing for more than 20 minutes | See [Troubleshoot data migration](/tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md#migration-errors-and-solutions) for help. |
```

File: tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md [hunk 12]
```diff
@@ -94,11 +94,26 @@ You can subscribe to TiDB Cloud alert emails to be informed in time when an aler
...
+
+- "Data migration job has been paused for more than 6 hours during incremental migration" 
+
+    Recommended action: resume the data migration job or ignore this alert.     
+
 - "Replication lag is larger than 10 minutes and stilling increasing for more than 20 minutes"
 
+    - Recommended action: contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for help.
+
+
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md:116]:
```suggestion
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**